invenio_subjects_mesh/cli.py

```diff
@@ -19,13 +19,16 @@
 
 
 @click.command()
-def main():
-    """Generate new subjects_mesh.yaml file."""
+@click.argument("year")
+def main(**parameters):
+    """Generate new subjects_mesh.jsonl file."""
+    year = parameters["year"]
+
     downloads_dir = Path(__file__).parent / "downloads"
     files = MeSHDownloader(downloads_dir)
-    files.download()
+    files.download(year)
```

Comment:
`year` is the name of the file here? Is this how it is provided from NLM(or whatever the source is), or do we have to do some renaming to the desired year?

Reply:
`year` is *part* of the name of the files to download: e.g., `d2022.bin` or `q2023.bin`. MeSHDownloader builds the URL based on that year.